ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: onsdigital/dp-concourse-tools-node-go
-    tag: latest
+    tag: 1.19.4-rc.1
```

Comment:
I don't think you should be using a release candidate
I've been using 1.19.2 elsewhere, which is good enough